### PR TITLE
Only note spec and generator version in gemspec file

### PR DESCRIPTION
**Pull request by bradymholt**
Rather than adding a comment on most generated files with the version of the spec and the version of the generator, we will note it once in the gemspec file.  This will reduce the diff in future PRs while still preserving the helpful version numbers.

### CODE REVIEW

File: ynab.gemspec [hunk 1]
```diff
@@ -23,7 +20,7 @@ Gem::Specification.new do |s|
   s.email       = ["api@ynab.com"]
   s.homepage    = "https://github.com/ynab/ynab-sdk-ruby"
   s.summary     = "YNAB API Endpoints Ruby Gem"
-  s.description = "Ruby gem wrapper for the YNAB API. Read the documentation at https://api.ynab.com"
+  s.description = "Ruby gem wrapper for the YNAB API. Read the documentation at https://api.ynab.com Generated from server specification version 1.70.0 using OpenAPI Generator version 7.6.0."
```
Comment (bradymholt) [inline ynab.gemspec:23]: Versions are now noted once, here in the gemspec.

File: templates/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This directory contains override templates for generating Ruby code from the OpenAPI spec.  The modify the default behavior or the generator.
```
Comment (bradymholt) [inline templates/README.md:1]: Added this README for future reference when we need to override a template.

File: templates/api_info.mustache [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{{#appName}}
```
Comment (bradymholt) [inline templates/api_info.mustache:1]: Overriding this template is the primary change here.  The original (default) version version cane be found here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/ruby-client/api_info.mustache

**Review by joshmenden — APPROVED** [review]
Love it

**Pull request merged**